modules/mobile-agents/android/client/client/src/main/java/org/wso2/emm/agent/AlreadyRegisteredActivity.java

```diff
@@ -111,8 +116,10 @@ protected void onCreate(Bundle savedInstanceState) {
 		btnUnregister = (Button) findViewById(R.id.btnUnreg);
 		btnUnregister.setTag(TAG_BTN_UNREGISTER);
 		btnUnregister.setOnClickListener(onClickListenerButtonClicked);
-		LocalNotification.startPolling(context);
-
+		if (devicePolicyManager.isAdminActive(cdmDeviceAdmin)) {
+			LocalNotification.startPolling(context);
```

Comment:
Ideally we should start/stop polling on Device Admin Enabled/Disabled states. So let's move start/stop polling to AgentDeviceAdminReciever's onEnabled() and onDisabled() methods.